phase2_stopvms.py

```diff
@@ -22,19 +22,17 @@
 # For those usages not covered by the Apache version 2.0 License please
 # contact with opensource@tid.es
 #
-author = 'chema'
-
 from os import environ as env
 import os
-import logging
 import cPickle as pickle
 import datetime
 
-from user_resources import UserResources
-from settings.settings import TRUSTEE
-import settings.settings
+from skuld.user_resources import UserResources
+import conf.settings
 import utils.log
 
+author = 'chema'
```

Comment:
I prefer to use native python object syntax for this type of variables, like this: \__author__